lua/nvim-treesitter/parsers.lua

```diff
@@ -7,14 +7,16 @@ list.javascript = {
   install_info = {
     url = "https://github.com/tree-sitter/tree-sitter-javascript",
     files = { "src/parser.c", "src/scanner.c" },
-  }
+  },
+  maintainers = {"@steelsojka"}
```

Comment:
Trailing comma for all new elements

```suggestion
  maintainers = {"@steelsojka"},
```

Reply:
fixed